packages/manager/apps/dedicated/client/app/account/user/user.controller.js

```diff
@@ -44,4 +55,12 @@ export default class UserAccountCtrl {
       }
     }.bind(this);
   }
+
+  descriptionOfHeading() {
+    const getRegion = this.coreConfig.getRegion();
```

Comment:
Same as here:
https://github.com/ovh/manager/pull/6688/files#r865634536

Reply:
It's applied